src/config/cambodia/layers.json

```diff
@@ -1642,6 +1642,11 @@
   "kh_incident_report": {
     "title": "Disaster impact report",
     "type": "point_data",
+    "admin_level_display": true,
+    "path": "https://prism-api.ovio.org/kobo/forms",
+    "adminCode": "Commune",
+    "adminLevel": 1,
+    "dataField": "NumPeoAff",
```

Comment:
We could probably simplify this and have something like this instead:

`admin_level_display: {adminCode: Commune, adminLevel: 1}`

Also, a lot of the config seems redundant betwee:
- path / data / date_url
- dataField / measure_field / feature_info_props / measure

Reply:
@ericboucher can you check if the current fix is sufficient enough?